src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/db/ScriptManagerImpl.java

```diff
@@ -151,16 +151,23 @@ private void executeScriptOneSwoop(
       Schema schema,
       DataEntityManager dataEntityManager)
       throws SQLException, IOException {
+    String fileSuffix = dataEntityManager.isResumable() ? "_temp.avro" : ".avro";
```

Comment:
".avro" as a string literal is used 4 times in the file. Consider making it `static final`
The same suggestion, but with the different justification, applies to "_temp.avro" – it's not a method specific and should be moved as well. Also, I expect it anyway will be needed for the recovery functionality

Reply:
Actually "_temp" or ".avro" will not be used individually for the recovery functionality (rather as part of a larger regex pattern) but I don't think it hurts to give them a special place.